crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -547,13 +533,21 @@ impl PrepassPipeline {
         // is enabled, the material uses alpha cutoff values and doesn't rely on the standard
         // prepass shader, or we are emulating unclipped depth in the fragment shader.
         let fragment_required = !targets.is_empty()
+            // PERF: This line forces the "prepass fragment shader" to always run in
```

Comment:
shouldn't that be above the line that pushes the shader def?

Reply:
Looking at the issue and where it was located before, this is relevant to unclipped depth rather than the shader def being pushed generally, which is why I moved it here.